### DISCUSSION TITLE
How do I embed multiple Jupyter-Notebooks in one Quarto File?

### DISCUSSION THREAD

**Discussion by JanBezw** [quarto-dev/quarto-cli#7554] [Q&A]
### Description

Hello,
i would like to embed multiple Jupyter-Notebooks in one .qmd-file. If i just write the embed commands below each other, only the last one will be exectued and displayed within my embed commands. Is there an option (or am I having a syntax error) for emebding multiple jupyter-Notebooks? Ive provided a MWE.

Kind regards,

Test.qmd
`
{{< embed Test1.ipynb echo=false >}} 
`
`
{{< embed Test2.ipynb echo=false >}} 
`

Test1.ipynb 
`
from sympy.abc import * 
`
`
from sympy.functions import *
`
`
from spb import plot
`
`
plot(x ** 2 + 2, (x, -5, 5), title = "A")
`
`
plot((x + 2) ** 2, (x, -7, 3), title = "B")
;
`

Test2.ipynb
`
from sympy.abc import *
`
`
from sympy.functions import *
`
`
from spb import plot
`
`
plot(x ** 3 + 2, (x, -5, 5), title = "C")
`
`
plot((x + 2) ** 5, (x, -7, 3), title = "D")
;
`

**Comment by mcanouil** [comment]
Hello, could you please use code blocks, etc. to format your post?
Also, please make a small and simple example.
